docs/content/sftp.md

```diff
@@ -155,6 +155,11 @@ Here are the command line options specific to this remote.
 
 Ask for the SFTP password if needed when no password has been configured.
 
+#### --ssh-path-override ####
+
+Override path used by SSH connection. Allows checksum calculation when
+SFTP and SSH paths are different.
+
```

Comment:
A specific example of how to configure the Synology NAS with this flag would be useful.  If we go for the config setting, then it could be a new section.